.azure-pipelines/pipelines.yml

```diff
@@ -59,34 +60,16 @@ stages:
               GCP_SERVICE_ACCOUNT_KEY: $(GcpServiceAccountKey)
             displayName: "Generate docs"
 
-          - script: ci/run_envoy_docker.sh 'ci/upload_gcs_artifact.sh /source/generated/docs docs'
+          - script: |
+              SHORT_COMMIT_SHA=$(git log --pretty=%P -n 1 | cut -d' ' -f2 | head -c7)
+              export SHORT_COMMIT_SHA
+              ci/run_envoy_docker.sh 'ci/upload_gcs_artifact.sh /source/generated/docs docs'
```

Comment:
Don't change this part, or at least in a separate PR. I find this frustrating because I can't just enter URL by the PR number to get docs preview. This SHA is the merge commit and not easy to figure out.

Reply:
> I find this frustrating because I can't just enter URL by the PR number to get docs preview

you can copy the sha immediately above the CI window that is what i am doing

i am literally using this all day and the current setup with PR is a massive regression. The caching alone makes it pretty unusable

i will also fix the repokitteh PR so we can do `/docs` to have the url  printed out

as a suggestion - how about using both. There is some convenience to having the latest  build on the PR number, altho i probably wont use it that often.

> This SHA is the merge commit and not easy to figure out.

no, its the commit head in the devs branch